lib/src/mamac_device.dart

```diff
@@ -51,9 +52,14 @@ class MamacDeviceNode extends SimpleNode {
       ..username = username
       ..password = password;
 
-    print('Type: $deviceType, Address: ${deviceAdress.toString()}, Refresh: $refresh');
+    print(
+        'Type: $deviceType, Address: ${deviceAdress.toString()}, Refresh: $refresh');
```

Comment:
Just an FYI, Generally prints are only used for debugging. If we have something we specifically want to log, should use:
`logger.warning` (or finest, fine, info etc) from the dslink package: `package:dslink/utils.dart`

I know there were some left-over debug statements still remaining when you started the project.